test/packages/other/readme_structure/_dev/build/build.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  ecs:
+    reference: git@1.10
+  docs_structure_enforced: true
```

Comment:
Maybe we can make this an object, so it doesn't need to be an all or nothing? This could allow for example to introduce some additional setting to force only a set of headers.
```
  docs_structure_enforced:
    enabled: true
    skip:
      - title: Performance and scaling
        reason: System metrics, no performance tuning available.
```